From phil@ipom.com Wed Jul 26 22:36:39 2006
Message-ID: <44C850DD.6090108@ipom.com>
Date: Wed, 26 Jul 2006 22:36:29 -0700
From: Phil Dibowitz <phil@ipom.com>
To: Ron Bouchard <bouchard.ron@gmail.com>, Greg KH <greg@kroah.com>
Cc: linux-usb-devel@lists.sourceforge.net,
	USB Storage list <usb-storage@lists.one-eyed-alien.net>
Subject: USB: unusual_devs device removal

This entry has been a mystery for some time. I had sent this patch as an
RFC a while ago, and now we've had two reports of this not being needed,
so I'm removing it.

In the event there are reports of breakage, we should revert this patch,
but add a US_FL_NEED_OVERRIDE flag.

Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/unusual_devs.h |   12 ------------
 1 file changed, 12 deletions(-)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h
@@ -627,18 +627,6 @@ UNUSUAL_DEV(  0x0595, 0x4343, 0x0000, 0x
 		"Digital Camera EX-20 DSC",
 		US_SC_8070, US_PR_DEVICE, NULL, 0 ),
 
-/* The entry was here before I took over, and had US_SC_RBC. It turns
- * out that isn't needed. Additionally, Torsten Eriksson
- * <Torsten.Eriksson@bergianska.se> is able to use his device fine
- * without this entry at all - but I don't suspect that will be true
- * for all users (the protocol is likely needed), so is staying at
- * this time. - Phil Dibowitz <phil@ipom.com>
- */
-UNUSUAL_DEV(  0x059f, 0xa601, 0x0200, 0x0200, 
-		"LaCie",
-		"USB Hard Disk",
-		US_SC_DEVICE, US_PR_CB, NULL, 0 ),
-
 /* Submitted by Joel Bourquard <numlock@freesurf.ch>
  * Some versions of this device need the SubClass and Protocol overrides
  * while others don't.
